RTT handler raises a TypeError from datetime

Using the `rtt_ms` handler results in a `TypeError`:

```
>       rtt = round((context.end - context.start) / timedelta(microseconds=1) / 1000)
E       TypeError: can't subtract offset-naive and offset-aware datetimes
```

`context.start` is created offset-aware while `context.end` is not for some reason
https://github.com/villainy/grpc-accesslog/blob/04096b0bd4f511477af1915dc4482f5647f1a9f8/src/grpc_accesslog/_server.py#L81-L83

--- a/src/grpc_accesslog/_server.py
+++ b/src/grpc_accesslog/_server.py
@@ -80,7 +80,7 @@ class AccessLogInterceptor(ServerInterceptor):
         """
         start = datetime.now(timezone.utc)
         response = method(request, context)
-        end = datetime.now()
+        end = datetime.now(timezone.utc)
 
         log_context = LogContext(
             context,

--- a/tests/test_server.py
+++ b/tests/test_server.py
@@ -1,10 +1,12 @@
 """Server interceptor tests."""
 import logging
+import re
 from unittest.mock import Mock
 
 from pytest import LogCaptureFixture
 
 from grpc_accesslog import AccessLogInterceptor
+from grpc_accesslog import handlers
 
 
 def test_default_handlers() -> None:
@@ -31,3 +33,21 @@ def test_intercept(caplog: LogCaptureFixture) -> None:
     interceptor.intercept(Mock(), Mock(), Mock(), "/abc/Test")
 
     assert "this that" in caplog.text
+
+
+def test_rtt_handler_succeeds(caplog: LogCaptureFixture) -> None:
+    """Test RTT handler should not raise errors through interceptor."""
+    caplog.set_level(logging.INFO, logger="root")
+
+    interceptor = AccessLogInterceptor(
+        name="root",
+        handlers=(
+            lambda _: "this",
+            handlers.rtt_ms,
+        ),
+        propagate=True,
+    )
+
+    interceptor.intercept(Mock(), Mock(), Mock(), "/abc/Test")
+
+    assert re.match(r"^.*this \d+", caplog.text)

